vita3k/main.cpp

```diff
@@ -126,10 +126,6 @@ int main(int argc, char *argv[]) {
     EmuEnvState emuenv;
     const auto config_err = config::init_config(cfg, argc, argv, root_paths);
 
-    // Create preference path for safety
-    if (!fs::exists(cfg.pref_path))
-        fs::create_directories(cfg.pref_path);
-
```

Comment:
Why deleted?

Reply:
Because it's done in init_paths() along with the other directories, which is called before this line.

Reply:
The idea of this PR https://github.com/Vita3K/Vita3K/pull/2989 is to move creation of pref_path after loading config. So it should not be done in init_paths()

Reply:
Yes, creating the pref path directory if it doesn't exist should only be done after the config file has been loaded in case it has been changed.

Reply:
for me, check if folder exist is only needed after config load cfg user pref path 🤔

Reply:
Fixed.

Reply:
I don't think root_paths is updated after the call to config::init, so you should use cfg.pref_path in this case.